og_ui/src/EventSubscriber/OgUiEventSubscriber.php

```diff
@@ -28,53 +29,55 @@ public static function getSubscribedEvents() {
    */
   public function provideDefaultOgPermissions(PermissionEventInterface $event) {
     $event->setPermissions([
-      'subscribe' => [
+      new GroupPermission([
+        'name' => 'subscribe',
         'title' => t('Subscribe to group'),
         'description' => t('Allow non-members to request membership to a group (approval required).'),
         'roles' => [OgRoleInterface::ANONYMOUS],
         'default roles' => [OgRoleInterface::ANONYMOUS],
-      ],
-      'subscribe without approval' => [
+      ]),
+      new GroupPermission([
+        'name' => 'subscribe without approval',
         'title' => t('Subscribe to group (no approval required)'),
         'description' => t('Allow non-members to join a group without an approval from group administrators.'),
         'roles' => [OgRoleInterface::ANONYMOUS],
-      ],
-      'unsubscribe' => [
+        'default roles' => [],
+      ]),
+      new GroupPermission([
+        'name' => 'unsubscribe',
         'title' => t('Unsubscribe from group'),
         'description' => t('Allow members to unsubscribe themselves from a group, removing their membership.'),
         'roles' => [OgRoleInterface::AUTHENTICATED],
-        'default roles' => [OgRoleInterface::AUTHENTICATED],
-      ],
-      'approve and deny subscription' => [
+      ]),
+      new GroupPermission([
+        'name' => 'approve and deny subscription',
         'title' => t('Approve and deny subscription'),
         'description' => t('Users may allow or deny another user\'s subscription request.'),
-        'default roles' => [OgRoleInterface::ADMINISTRATOR],
-      ],
-      'add user' => [
+      ]),
+      new GroupPermission([
+        'name' => 'add user',
         'title' => t('Add user'),
         'description' => t('Users may add other users to the group without approval.'),
-        'default roles' => [OgRoleInterface::ADMINISTRATOR],
-      ],
-      'manage members' => [
+      ]),
+      new GroupPermission([
+        'name' => 'manage members',
         'title' => t('Manage members'),
         'description' => t('Users may remove group members and alter member status and roles.'),
-        'default roles' => [OgRoleInterface::ADMINISTRATOR],
```

Comment:
I'm now doubting if it is a good idea to assign the permission by default to the administrator. While this is certainly the most common case, this might be unexpected for people that are implementing permissions but aren't familiar with it.